### ISSUE TITLE
// eslint-ignore-next-line not working outside style tag

### ISSUE THREAD

**Issue by Zzzen** [vuejs/eslint-plugin-vue#1497]
**Checklist**

- [x] I have tried restarting my IDE and the issue persists.
- [x] I have read the [FAQ](https://eslint.vuejs.org/user-guide/#faq) and my problem is not listed.

**Tell us about your environment**

- **ESLint version:*7.22.0* 
- **eslint-plugin-vue version:*7.2.0* 
- **Node version:*15.6.1* 
- **Operating System:*windows 10* 

**Please show your full configuration:**

```js
module.exports = {
    extends: [
        require.resolve('./privateConfig'),
        'plugin:vue/recommended',
        'plugin:vue-scoped-css/recommended',
    ],
    rules: {
        'vue-scoped-css/require-scoped': 'error',
    }
```

**What did you do?**	

```vue
// eslint-disable-next-line vue-scoped-css/require-scoped
<style>
  body {}
</style>
```

**What did you expect to happen?**
`// eslint-ignore-next-line` should work. No error should be reported.

**What actually happened?**

"Missing `scoped` attribute.eslintvue-scoped-css/require-scoped" is reported

**Repository to reproduce this issue**
currently no. I can create one if necessary.

![eslint-disable-vue](https://user-images.githubusercontent.com/6630042/119673033-a42c2000-be6d-11eb-92dd-b04e623ab9b4.gif)

**Comment by ota-meshi** [comment]
`// ...` is not a comment. We do not support it.
You should use HTML comments.
https://eslint.vuejs.org/user-guide/#disabling-rules-via-eslint-disable
